exercicios/para-casa/xenia-barreto/abilities.js

```diff
@@ -0,0 +1,13 @@
+function Ability(name, summary , effect){
+
+    this.name = name;
+    this.summary = summary;
+    this.effect = effect;
+}
+
+module.exports = Ability
+
+
```

Comment:
Oii Xenia, 
muito bom que separou as funções em arquivos diferentes.

Uma sugestão de como fiz no meu, criei uma variavél com o objeto das habilidades desse jeito: 
`const abilities  = {
    overgrow:
    {
      name: 'Overgrow',
      summary: 'Strengthens Grass moves to inflict 1.5× damage at 1/3 max HP or less.',
      effect: 'When this Pokémon has 1/3 or less of its HP remaining, its Grass-type moves inflict 1.5× as much regular damage.'
    }`

e depois quando instanciamos o novo objetos eu pego as propriedades desse objeto:
`const abilityOvergrow = new ability('Overgrow', abilities.overgrow.summary, abilities.overgrow.effect )`

É uma outra forma de fazer.

Reply:
@GabiTavaresV mulher eu vi isso depois em um review, muito inteligente!!!
Talvez eu pudesse ter chego a esse ponto, se tivesse conseguido me dedicar mais essa semana, infelizmente foi pessimo.
Vou consultar sua ideia sim, se me permitir 😄 
Obrigada pelo Review